src/lib/Tests/Form/Data/ContentTypeDataTest.php

```diff
@@ -23,52 +23,99 @@ public function testContentTypeDraft()
 
     public function testFieldDefinitionData()
```

Comment:
```suggestion
    public function testFieldDefinitionData(): void
```

Reply:
Technically was left as-is probably because it was already present. Considering that tests are not covered by BC promises, I'm adjusting this file.

Fixed in e6a2a544.